ci-operator/step-registry/baremetalds/assisted/install/baremetalds-assisted-install-workflow.yaml

```diff
@@ -1,6 +1,7 @@
 workflow:
   as: baremetalds-assisted-install
   steps:
+    cluster_profile: packet-assisted
     allow_best_effort_post_steps: true
```

Comment:
maybe linter will yell at the order here, let's see

Reply:
Ironically it didn't.. lol but the jobs all got the `cluster_profile` removed. 🤔